README.md

```diff
@@ -255,3 +255,9 @@ By default this example setup comes bundled with a number of dashboards, such as
   https://grafana.com/grafana/dashboards/16927-akeyless-gateway
 
   Secrets metrics requires CipherTrust Manager version 2.15.0 or higher.
+
+ - CipherTrust Active TCP connections
```

Comment:
All other dashboards have "CipherTrust Manager" and not just "CipherTrust". Can you please change it to make it consistent?